.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectCodeStyleSettingsManager">
+    <option name="PER_PROJECT_SETTINGS">
+      <value>
```

Comment:
Not again!? They ARE I’m my .gitignore…along with build/ and *.pyc. I’ll take a look (and also commit .gitignore).
> On May 30, 2015, at 9:43 PM, William Schoenell <notifications@github.com> wrote:
> 
> In .idea/codeStyleSettings.xml <https://github.com/astroufsc/python-si-tcpclient/pull/1#discussion_r31385777>:
> 
> > @@ -0,0 +1,13 @@
> > +<?xml version="1.0" encoding="UTF-8"?>
> > +<project version="4">
> > +  <component name="ProjectCodeStyleSettingsManager">
> > +    <option name="PER_PROJECT_SETTINGS">
> > +      <value>
> Add .idea to the .gitignore and remove these files.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/astroufsc/python-si-tcpclient/pull/1/files#r31385777>.
>